RHEL/6/input/system/software/integrity.xml

```diff
@@ -176,9 +176,9 @@ have file hashes different from what is expected by the RPM database.
 The hashes of important files like system executables should match the
 information given by the RPM database. Executables with erroneous hashes could
 be a sign of nefarious activity on the system.</rationale>
-<ident cce="27223-7"  stig="RHEL-06-000519" />
+<ident cce="27223-7" stig="RHEL-06-000519" />
 <oval id="rpm_verify_hashes" />
-<ref nist="CM-6(d),SI-7" disa="1496" />
+<ref nist="CM-6(d),SI-7" disa="366" />
```

Comment:
So here is what's happening. The script utilizes
input/auxiliary/stig_overlay.xml as a sort of legend mapping rule id to
stig id. The input/auxiliary/stig_overlay.xml file currently
references RHEL-06-000519
for rule id rpm_verify_hashes. Its doing what its suppose to. Perhaps in
this case, the input/auxiliary/stig_overlay.xml document should be updated
to reflect the correct STIG requirement for that rule. Looking at the
RHEL-06-00281
stig id in input/auxiliary/stig_overlay.xml shows no rule is currently
associated with it.

On Wed, Feb 25, 2015 at 3:39 PM Shawn Wells <notifications@github.com>
wrote:

> In RHEL/6/input/system/software/integrity.xml
> <https://github.com/OpenSCAP/scap-security-guide/pull/461#discussion_r25381750>
> :
>
> >  <oval id="rpm_verify_hashes" />
> > -<ref nist="CM-6(d),SI-7" disa="1496" />
> > +<ref nist="CM-6(d),SI-7" disa="366" />
>
> there's something funky happening in the script. This rule maps to
> CCI-001496, which is RHEL-06-00281 in DISA's content (not RHEL-06-000519).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSCAP/scap-security-guide/pull/461/files#r25381750>
> .
>